scripts/train.py

```diff
@@ -178,7 +178,7 @@ def processThread(self):
                 print('Saving model weights has been cancelled!')
             exit(0)
         except Exception as e:
-            print(e)
+            raise e
```

Comment:
Somehow I agree that the print(e) is not convenient as it make a more readable error message but that masks the details. However I would be prefere this to be made in another PR